docs/content/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference.mdx

```diff
@@ -200,12 +200,26 @@ user_code_launcher:
   </ReferenceTableItem>
   <ReferenceTableItem
   propertyName="config.server_process_startup_timeout">
-  The amount of time, in seconds, to wait for code to import when launching a new service for a code location. If your code takes an unusually long time to load and results in timeouts in the <strong>Deployment</strong> tab, you can increase this setting above the default. <strong>Note</strong> This setting isn't applicable to the time it takes for a job to execute.
+  The amount of time, in seconds, to wait for code to import when launching a new service for a code location. If your code takes an unusually long time to load after your ECS task starts up and results in timeouts in the <strong>Deployment</strong> tab, you can increase this setting above the default. <strong>Note</strong> This setting isn't applicable to the time it takes for a job to execute.
   <ul>
   <li><strong>Default</strong> - 180 (seconds)</li>
   </ul>
   </ReferenceTableItem>
   <ReferenceTableItem
+  propertyName="config.ecs_timeout">
+  How long (in seconds) to poll against ECS API endpoints. If your ECS tasks take an unusually long time to start and result in timeouts, you can increase this setting above the default.
```

Comment:
Maybe something that specifically mentions code servers? "How long (in seconds) to wait for ECS to spin up a new service and task for a code location"